### DISCUSSION TITLE
Ramblings of a price action trader with user-drawn trend lines

### DISCUSSION THREAD

**Discussion by arenddeboer** [Mathieu2301/TradingView-API#146] [Ideas]
I'm using this library to fetch my trend lines from TradingView and make sure I have a limit order at the current time/price intersection.
In an ideal world, I would not fetch my drawings every 10 seconds but receive a websocket message the drawings have changed. Not sure if this is possible with the lib. Currently using a Firefox extension to detect a chart save action and notify my backend this way. Obviously this doesn't work when using the TradingView mobile app to update trend lines. 
As far as I know, I cannot programmatically draw order positions, stop loss and take profit lines.
Basically I need access to TradingView's private broker API..

Anyone with a similar setup ?
